main.js

```diff
@@ -0,0 +1,107 @@
+// Add event listener 
+// prevent default and const for inputs
+document.getElementById('form-post').addEventListener('submit', function(e) {
+  e.preventDefault();
+
+ 
+  const textFromPost = document.getElementById('text-from-post').value;
+  const writerPosted = document.getElementById('writer-posted').value;
+  
+  postToAdd(textFromPost, writerPosted);
+
+  
+  document.getElementById('text-from-post').value = '';
+  document.getElementById('writer-posted').value = '';
+});
+
+// Fxn to make post add to page html 
+function postToAdd(post, writer) {
+  
+  const containerOfMessages = document.createElement('div');
+  containerOfMessages.classList.add('mb-4', 'border', 'p-3');
+
+  
+  const messagesPost = document.createElement('p');
+  messagesPost.textContent = `${post} - Posted by ${writer}`;
+
+  
+  const withdrawMessagesHere = document.createElement('a');
+  withdrawMessagesHere.href = '#';
+  withdrawMessagesHere.textContent = 'remove';
+  withdrawMessagesHere.classList.add('text-danger', 'mr-2');
+  withdrawMessagesHere.addEventListener('click', function () {
+    containerOfMessages.remove();
+  });
+
+  
+  const postSwitchFromPage = document.createElement('a');
```

Comment:
same as above, this const name is confusing. 
showHideCommentsAction or something is better.